packages/react/src/components/select/Select.module.scss

```diff
@@ -291,6 +326,65 @@
   }
 }
 
+.tagListContainer {
+  display: flex;
+  flex-direction: column;
+
+  .tagList {
+    display: flex;
+    flex-flow: row wrap;
+    max-height: 78px;
```

Comment:
Why 78px?

Reply:
It's the space needed to show two rows of tags. There is no constant for this, so have to hard code it.